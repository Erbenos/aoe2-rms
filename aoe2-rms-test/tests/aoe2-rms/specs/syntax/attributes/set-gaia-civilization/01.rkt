#lang aoe2-rms
; Create a Lithuanian monument for gaia somewhere on the map

(set-gaia-civilization 35)

<OBJECTS-GENERATION>
(create-object 'MONUMENT)
