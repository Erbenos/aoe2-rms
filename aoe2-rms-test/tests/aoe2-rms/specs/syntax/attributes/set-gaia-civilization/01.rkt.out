set_gaia_civilization 35
<OBJECTS_GENERATION>
create_object MONUMENT {
}
